test/datahike/test/query.cljc

```diff
@@ -117,24 +117,24 @@
                (d/db-with [{:db/id 1, :name "Ivan", :age 15}
                            {:db/id 2, :name "Petr", :age 37}
                            {:db/id 3, :name "Ivan", :age 37}]))]
-    (testing "Relation binding"
```

Comment:
Why are these tests (and others) commented out?

Reply:
just forgot to uncomment